roles/openshift_manageiq/vars/main.yml

```diff
@@ -22,3 +22,4 @@ manage_iq_tasks:
     - policy add-role-to-user -n management-infra management-infra-admin -z management-admin
     - policy add-cluster-role-to-user cluster-reader system:serviceaccount:management-infra:management-admin
     - policy add-scc-to-user privileged system:serviceaccount:management-infra:management-admin
+    - policy add-cluster-role-to-user system:image-puller system:serviceaccount:management-infra:management-admin
```

Comment:
It seems like you might want to start subdividing the power of your service accounts.  There's probably no reason for the pod running openscap to be able to read every deploymentconfig in the system, right?